_includes/figure

```diff
@@ -27,10 +27,10 @@ because https://github.com/jekyll/jekyll/issues/2248.
 
 <div class="figure-body">
 
-<div class="figure-images contains-{{ number-of-images }}{% endif %}">
+<div class="figure-images contains-{{ number-of-images }}">
     {%for image in images %}
         {% if include.link %}<a href="{{ include.link }}">{% elsif site.output == "web" %}<a href="{{ path-to-book-directory }}/{{ site.image-set }}/{{ image }}">{% else %}{% endif %}
```

Comment:
@SteveBarnett Ah, yes, that `{% else %}` can come out. Fixed in 5eb42b8.

The `if`s do need to wrap the `a` because if there is no link and this is not web output, we must not have the link at all. The main reason for this is that in an epub (EPUB2 at least, haven't checked EPUB3) we cannot link to an image file, only to (X)HTML files.